packages/webapp/src/routes/developer/routing-result.js

```diff
@@ -2,7 +2,7 @@ import constants from '../../utils/constants.js'
 
 const handlers = {
   get: async (_request, h) => h.view(constants.views.DEVELOPER_ROUTING_RESULT),
-  post: async (_request, h) => h.redirect(constants.routes.DEVELOPER_ELIGIBILITY_ENGLAND)
+  post: async (_request, h) => h.redirect(constants.routes.DEVELOPER_ROUTING_RESULT)
```

Comment:
@paulsimonandrews - do you know what this route is? I just put it to redirect to itself but where should it redirect to?

Reply:
I think this is fine for the moment. These routes look like they aren't needed any more anyway so will need to removed, but that isn't in scope here.